docs/src/components/FullCodeOnGithubButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import GitHubIcon from '@mui/icons-material/GitHub';
+
+interface FullCodeOnGithubProps {
+    url: string;
+}
+
+export const FullCodeOnGithub: React.FC<FullCodeOnGithubProps> = (props) => {
+    const { url, ...otherButtonProps } = props;
+
+    const release =
+        'https://github.com/brightlayer-ui/react-component-library/blob/feature/3290-full-code-on-gh-poc/docs/src';
+    return (
+        <Button
+            {...otherButtonProps}
+            sx={{ ml: 2 }}
```

Comment:
Should this be a default style? What if you want to use it in a different location without the marginLeft